mon-pix/translations/fr.json

```diff
@@ -90,6 +90,11 @@
     "skip-links": {
       "skip-to-content": "Aller au contenu",
       "skip-to-footer": "Aller au bas de page"
+    },
+    "data-protection-policy-information-banner" : {
+      "url-label": "Politique de protection des données.",
+      "title": "Notre politique de confidentialité a évolué. Nous vous invitons à en prendre connaissance : ",
+      "aria-label": "Bannière d'information sur la politique de confidentialité Pix"
```

Comment:
**question**
Est-ce que cet aria label apporte vraiment une information pertinente ? J'ai l'impression qu'on pourrait s'en passer.

Je lis ici (https://hmh.engineering/accessibility-is-hard-careful-with-them-aria-labels-5e1c6c669b61) qu'il ne faut pas utiliser `aria-label` pour apporter une information supplémentaire parce qu'il remplace le contenu de l'élément. Éventuellement on peut utiliser `aria-description` pour apporter une précision. Mais je ne suis pas sûr que ce soit utile ici, l'info qu'on veut transmettre c'est que la politique de confidentialité a été mise à jour.

Reply:
Honnêtement c'est le test qui a influencé ce choix d'ajouter un arial-label aux bannières.
Si on veut tester la présence de la bannière avec ByRole, testing library la retrouve (avec en nom accessible 'alert') mais sans nom.
Ce qui me permet de faire :
```
assert.dom(screen.getByRole('alert', {
                name: this.intl.t('common.communication-banner.aria-label'),
              })).exists();
```

Autrement je peut juste faire un getByRole('alert') et un ByText avec le contenu de la bannière.

Reply:
Oui ce serait mieux de faire autrement si c'est possible, on ne devrait pas ajouter du code juste pour les tests, surtout du code qui potentiellement casse l'accessibilité !